CheatSheets/windows-cheatsheet.md

```diff
@@ -42,9 +42,16 @@
 
 ## File Explorer Shortcuts
 
-| Command | Description |
-| ------- | ----------- |
-| `Ctrl + W` | Close the current window |
+| Command | Description                 |
+|---------|-----------------------------|
+| `Ctrl + W` | Close the current window    |
+| `Shift + Tab`| Set Focus on Side Bar       |
+| `Ctrl + N`| Duplicate the File Explorer |
+| `Crtl + Shift + N`| Create a New Folder         |
+| `Alt + ←`| Navigate to previous folder |
+| `Ctrl + L` | Navigate to the address bar |
```

Comment:
These commands are already merged via your first PR. Please, remove your `windows-cheatssheet.md` file's modification and only commit for `nvm-cheatsheet.md` file. @SaumyaBhushan